### DISCUSSION TITLE
Any one-line solution to adjust component font size according to the viewport size?

### DISCUSSION THREAD

**Discussion by Trav1sRen** [mantinedev/mantine#6294] [Q&A]
I wanna find a way to most conveniently change the component font size **globally** according to the viewport size.
Instead of customize each component using Media Query or `font-size: clamp(1.5rem, 2.5vw, 4rem); `, is there any cleaner solution?

**Comment by rtivital** [comment]
If you want to apply font-size to all elements on the page, use `* { font-size: clamp(1.5rem, 2.5vw, 4rem) !important;  }`
